packages/did-ethr-resolver/src/implementations/ewJsonRcpSigner.ts

```diff
@@ -0,0 +1,20 @@
+import { providers } from 'ethers';
+import { EwSigner } from './ewSigner';
+
+/**
+ * Example:
+ * import detectMetamask from "@metamask/detect-provider";
+ * const web3Provider = await detectMetamask()
+ * const web3IdentityOwner = Web3IdentityOwner.fromWeb3Provider(web3Provider)
+ * @param web3Provider: an EIP1193 provider
+ */
+type Web3ProviderType = providers.ExternalProvider | providers.JsonRpcFetchFunc;
+
+export class EwJsonRpcSigner extends EwSigner {
+    publicKey: string | undefined;
```

Comment:
Why have the `publicKey` property here? I think we have `IdentityOwner` to associate the publicKey with signer
```suggestion
```

Reply:
I cannot remember why I put it here, i don't even use it. I agree, we just need publickey to be on IdentityOwner. Thank you John